src/models/FGAuxiliary.cpp

```diff
@@ -180,7 +184,15 @@ bool FGAuxiliary::Run(bool Holding)
   vMachUVW(eV) = vAeroUVW(eV) / in.SoundSpeed;
   vMachUVW(eW) = vAeroUVW(eW) / in.SoundSpeed;
 
-  // Position
+  // Position tracking in local frame with local frame origin at lat, lon of initial condition
+  // and at 0 altitude relative to the reference ellipsoid. Position is NEU (North, East, UP) in feet.
+  if (!NEUFromStartInitialized) {
+    NEUStartLocation = FDMExec->GetIC()->GetPosition();
+    NEUStartLocation.SetPositionGeodetic(NEUStartLocation.GetLongitude(), NEUStartLocation.GetGeodLatitudeRad(), 0.0);
+    NEUFromStartInitialized = true;
+  }
```

Comment:
I'd rather initialize `NEUStartLocation` in a method that would be called by `FGFDMExec::Initialize`:
https://github.com/JSBSim-Team/jsbsim/blob/83e3df2c49c13eae78ab7853848cdd9e02c6c5ac/src/FGFDMExec.cpp#L676-L681
Since `FGFDMExec::Initialize` is called itself by `FGFDMExec::RunIC` that would ensure that `NEUStartLocation` would be initialized at the same time than the rest of the simulation. And that would save te usage of the flag `NEUFromStartInitialized`.